DriveClass/src/drive.cpp

```diff
@@ -0,0 +1,107 @@
+/*----------------------------------------------------------------------------*/
+/*                                                                            */
+/*    Module:       drive.cpp                                                  */
+/*    Author:       Alex Mei                                                  */
+/*    Created:      Sat Feb 22 2020                                           */
+/*    Description:  V5 project                                                */
+/*                                                                            */
+/*----------------------------------------------------------------------------*/
+
+// ---- START VEXCODE CONFIGURED DEVICES ----
+// ---- END VEXCODE CONFIGURED DEVICES ----
+
+#include "vex.h"
+
+using namespace vex;
+
+
+class Drive{
+  public:
+    static constexpr double PI = 3.14159265;
```

Comment:
Why did you rewrite your pi variable in both .cpp and .h, you only need it in one.